src/utils/befristung.ts

```diff
@@ -0,0 +1,110 @@
+import { watch, type ComputedRef, type Ref } from 'vue';
+import { getNextSchuljahresende } from './date';
+import type { useForm } from 'vee-validate';
+import { useRollen, type TranslatedRolleWithAttrs } from '@/composables/useRollen';
+import { RollenMerkmal } from '@/stores/RolleStore';
+
+export enum BefristungOption {
```

Comment:
Kann funktionalität von hier in das BefristungsInput verschoben werden, sodass für die übergeordneten Komponenten weniger zu orchestrieren ist?

Reply:
Done. Der Parent jetzt nimmt nur der finale String und muss nichts anders machen als daraus ein Date zu machen nach der Validierung.